test/integration/tree_test.go

```diff
@@ -20,6 +20,8 @@ package integration
 
 import (
 	"context"
+	"github.com/yorkie-team/yorkie/api/converter"
```

Comment:
Please place the packages into 3 groups.

- default packages
- 3rd party packages
- internal packages